performance-tools/benchmark.md

```diff
@@ -0,0 +1,56 @@
+# Computer Vision Pipeline Benchmarking
+
+The provided Python-based script works with Docker Compose to get pipeline performance 
+metrics like video processing in frames-per-second (FPS), memory usage, power
+consumption, and so on.
+
+# Prerequisites 
+
+- Python environment v3.12.2
+  - could use [Miniconda](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-python.html) and create a [Python 3.12.2 env](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-python.html)
+- Python packages listed in requirements.txt
```

Comment:
can we add some steps to install these packages? , the requirements.txt is not in an intuitive path. I believe is somewhere under `./docker/platform/`